Client/Frontend/Fakespot/Views/AdjustRatingView.swift

```
@@ -0,0 +1,142 @@
+    private lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.titleFontSize,
+                                                            weight: .medium)
+        label.numberOfLines = 0
+    }
+
+    private lazy var descriptionLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.descriptionFontSize,
+                                                            weight: .regular)
+        label.numberOfLines = 0
+    }
+
+    private lazy var starRatingView: StarRatingView = .build()
+
+    private lazy var hStackView: UIStackView = .build { stackView in
+        stackView.axis = .horizontal
+        stackView.spacing = UX.hStackSpacing
+    }
+
+    private lazy var vStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.vStackSpacing
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UX.margins
+    }
+
+    private lazy var spacer: UIView = .build()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    func configure(_ viewModel: AdjustRatingViewModel) {
+        titleLabel.text = viewModel.title
+        titleLabel.accessibilityIdentifier = viewModel.titleA11yId
+
+        descriptionLabel.text = viewModel.description
+        descriptionLabel.accessibilityIdentifier = viewModel.descriptionA11yId
+
+        let cardModel = ShadowCardViewModel(view: vStackView, a11yId: viewModel.cardA11yId)
+        cardContainer.configure(cardModel)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
```

This should throw a fatal error as mentioned [here](https://github.com/mozilla-mobile/firefox-ios/pull/16063#discussion_r1295843346).

---

fatalError it is